### PR TITLE
Added salt-api support

### PR DESCRIPTION
**Pull request by aboe76**
Add salt-api to salt-formula

### CODE REVIEW

**Comment by nmadhok** [comment]
@aboe76 can you add ------ under salt.api

File: README.rst [hunk 1]
```diff
@@ -41,6 +41,11 @@ Install salt cloud.
 Install salt-ssh with roster file.
 Configure pillar data under salt:ssh_roster to feed the template.
 
+``salt.api``
+
```
Comment (nmadhok) [inline README.rst]: Add ``-----------`` under salt.api

**Comment by aboe76** [comment]
reformatted sorry, forgot to add ----------- it's late here :)

**Comment by nmadhok** [comment]
@aboe76 no problem! Thanks for adding in the api stuff.

**Comment by aboe76** [comment]
was the only thing missing in the formula I think...

**Pull request merged**